setup.py

```diff
@@ -59,7 +59,7 @@ def read_file(filename, encoding='utf8'):
 # So this dependency is not added for old Linux distributions with old setuptools,
 # in order to allow these systems to build certbot from sources.
 if StrictVersion(setuptools_version) >= StrictVersion('36.2'):
-    install_requires.append("pywin32 ; sys_platform == 'win32'")
+    install_requires.append("pywin32>=224 ; sys_platform == 'win32'")
```

Comment:
I hate that our oldest unit tests pass, but I ran our nightly integration tests with this PR and the oldest tests failed due to this value of `install_requires`. See https://travis-ci.com/certbot/certbot/builds/122170998.

Reply:
I do not understand what is going on here. Why specifying a version is wrong here?

Reply:
I don't immediately know either. I suspect the problem is we need an even newer version of setuptools. Alternatively, you can't specify both a version and a sys_platform requirement, but I think/hope this isn't it.

Reply:
I suppose it could also be a syntax problem. Maybe you have to specify versions slightly differently if it also has the `sys_platform` stuff?

Reply:
Yes but in this case, why it is passing with more recent versions of setuptools?

Reply:
Alternatively, we can just keep this dependency uncapped in setup.py. I added it to be consistent with the dev constraint, but in fact, older versions of pywin32 are working correctly.

Reply:
I got it! It is not because of this line, it is because I added `pywin32` into the `setup.py` of Certbot-CI, but we are using setuptools 1.0.0, and so env markers are not supported.

We need to do the same conditional append depending on setuptools version than in the setup.py of `certbot`.

Reply:
I updated the `setup.py` of certbot-ci, and rebased the branch `test-boulder-integration` on top of `fix-integration-tests-windows`. The relevant travis build is: https://travis-ci.com/certbot/certbot/builds/122180642

Reply:
Ah that makes more sense. When I was reviewing, I was just looking at the diff from the last review and didn't see `certbot-ci/setup.py` change here.

I don't think it's going to work though due to a problem that's also in Certbot's `setup.py` that wasn't introduced in this PR. What I think is going to happen is `setup.py` will run with an old version of `setuptools` and the requirement of `pywin32` is never added.

Reply:
Well, I think the problem I described still exists but tests didn't fail because we never run on AppVeyor with an old version of setuptools. We can probably fix this problem in both Certbot and certbot-ci in another PR.

Reply:
Would you expect me to do things half way at this point on Windows ;)? I used the very same approach on certbot-ci than on certbot: pywin32 is not required on Linux, in fact, its absence trigger the logic specific to Linux.

Reply:
Concerning setuptools version, I will answer on the specific issue you created for that, but short answer is: running tests on old versions of setuptools will be irrelevant.